help/destinations/catalog/social/tiktok.md

```diff
@@ -0,0 +1,114 @@
+---
+title: TikTok connection // Replace with the name of destination
+description: Create segments from your customer lists from people who've visited your site or people who have already interacted with your content on TikTok and quickly push the desired segment to TikTok using TikTok's real-time streaming API to use for ad targeting. // Replace with the paragraph in the Overview section
```

Comment:
Same comment as above, please remove the trailing part of the sentence ` // Replace with the paragraph in the Overview section`